model/config.go

```diff
@@ -1037,6 +1038,7 @@ type SupportSettings struct {
 	ReportAProblemLink          *string
 	SupportEmail                *string
 	CustomTermsOfServiceEnabled *bool
+	ReAcceptancePeriod          *int
```

Comment:
Can we name this `CustomTermsOfServiceReAcceptancePeriod`, so it's consistent with other custom terms of service settings?

Reply:
Please also help add to ../app/diagnostics.go

Reply:
1. Renamed `ReAcceptancePeriod` to `CustomTermsOfServiceReAcceptancePeriod`.
1. Added `CustomTermsOfServiceReAcceptancePeriod` to app diagnostics.